test_out_table.ext

```diff
@@ -0,0 +1 @@
+LCSC Part,First Category,Second Category,MFR.Part,Package,Solder Joint,Manufacturer,Library Type,Description,Datasheet,Stock,Price

```

Comment:
What's the purpose of this file?

Reply:
It was a mistake, good catch, thanks. Didn't realize it got tracked.